### PR TITLE
Issue #6488: Use CSS property for admin bar height.

### PR DESCRIPTION
**Pull request by quicksketch**
Amend @indigoxela's PR for https://github.com/backdrop/backdrop-issues/issues/6488.

### CODE REVIEW

File: core/modules/admin_bar/css/admin_bar.css [hunk 1]
```diff
@@ -6,6 +6,9 @@
  *
  * @see https://www.htmldog.com/articles/suckerfish
  */
+:root {
+  --admin-bar-height: 2.25em;
```
Comment (indigoxela) [inline core/modules/admin_bar/css/admin_bar.css:10]: We can't use "em" in this case, as any place, where the var is used may have a different value for that (inherited, or set). That means (according to my testing), the value's different depending on context.

Example: grab a heading from content, apply the var for positioning -> looks odd, as the heading has a bigger em value.

Comment (indigoxela) [reply core/modules/admin_bar/css/admin_bar.css:10]: We *could* switch to `rem`, but also in the whole admin-bar CSS, which requires a bigger change.

Comment (indigoxela) [reply core/modules/admin_bar/css/admin_bar.css:10]: The required change might not be that big, though, as currently everything inside admin-bar's already based on the base font size (12px).

So, if we made it:
font: normal .75rem "Lucida Grande", Verdana, sans-serif;

and then, based on that:
--admin-bar-height: 2.35rem;

... we'd use the same base unit for calculations...

Comment (quicksketch) [reply core/modules/admin_bar/css/admin_bar.css:10]: Thanks @indigoxela, insightful. I still haven't got a full grasp on em/rem.

**Pull request merged**